yb-voyager/src/srcdb/postgres.go

```diff
@@ -392,13 +392,20 @@ func (pg *PostgreSQL) getExportedColumnsListForTable(exportDir, tableName string
 // Given a PG command name ("pg_dump", "pg_restore"), find absolute path of
 // the executable file having version >= `PG_COMMAND_VERSION[cmd]`.
 func GetAbsPathOfPGCommandAboveVersion(cmd string, sourceDBVersion string) (path string, binaryCheckIssue string, err error) {
+	// In case of pg_dump and pg_restore min required version is the max of min supported version of the cmd and sourceDBVersion
+	// In case psql min required version is the min supported version of psql
+	minRequiredVersion := max(PG_COMMAND_VERSION[cmd], sourceDBVersion)
```

Comment:
let's change the `version.CompareSimple` logic down below also to use the `minRequiredVersion` ?